ERCS/erc-7599.md

```diff
@@ -0,0 +1,132 @@
+---
+eip: 7599
```

Comment:
Please note EIP/RIP/ERC numbers are assigned by editors and associates.  Number sniping is not allowed.

As 7599 is the next sequential number and there isn't apparent number sniping I will assign as is.